src/main/java/service/WeatherData.java

```diff
@@ -1,4 +1,4 @@
 package service;
 
-public record WeatherData(double temperature) {
+public record WeatherData(double temperature, boolean precipitation) {
```

Comment:
I think you would also need to change the name of this variable for it to not cause merge conflicts

Reply:
yep